### PR TITLE
feat(cmd): bucket create to accept org name as flag

### LINKED ISSUE #15828: influx bucket create cannot use organization name
The `influx bucket create` command doesn't seem to have a `-o` flag for specifying the organization name. Here is what I did on my local box using the latest master at revision a64b976561c798f9aeeb771c4f48fb5d3141ad5f.

```
jsternberg:[~/g/s/g/i/influxdb] $ (master $=) influx bucket create -o influxdata -n NOAA                                              [7/7]
Error: unknown shorthand flag: 'o' in -o
Usage:
  influx bucket create [flags]

Flags:
  -h, --help                 Help for the create command
  -n, --name string          Name of bucket that will be created
      --org-id string        The ID of the organization that owns the bucket
  -r, --retention duration   Duration in nanoseconds data will live in bucket

Global Flags:
      --host string    HTTP address of Influx (default "http://localhost:9999")
      --local          Run commands locally against the filesystem
  -t, --token string   API token to be used throughout client calls

jsternberg:[~/g/s/g/i/influxdb] $ (master $=) influx org find
ID                      Name
97aa81cc0e247dc4        influxdata
jsternberg:[~/g/s/g/i/influxdb] $ (master $=) influx bucket create --org-id 97aa81cc0e247dc4 -n NOAA
ID                      Name    Retention       Org     OrganizationID
b76b31ef2f0341e0        NOAA    0s              <nil>   <nil>
```

Additionally, I found it weird that when I did create the bucket, the `Org` and `OrganizationID` fields were nil. It seems like these aren't getting copied in to the command's display. When I do `influx bucket find` these fields are still nil.

Most other commands support `-o org` as an option and it is a lot more convenient than having to find the organization id explicitly. It would also make writing tutorials easier because then you don't have to teach the person how to retrieve the organization id.

### PR DESCRIPTION
**Pull request by dearyhud**
enables a user to specify an organization name when creating a bucket from the influx cli.

Closes #15828

The influx bucket create command doesn't seem to have a -o flag for specifying the organization name. This PR adds a flag `--org` or `-o` for specifying the org name to avoid the overhead of having to lookup an organization id

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [x] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by jsteenb2** [comment]
can we add the tests in this body of work? a promise to come back and add tests in later, almost never comes to fruition. Be good to start delivering on our testing for cli stuff

File: cmd/influx/inspect.go [hunk 1]
```diff
@@ -63,6 +63,7 @@ in the following ways:
 	inspectReportTSMCommand.Flags().BoolVarP(&inspectReportTSMFlags.detailed, "detailed", "", false, "emit series cardinality segmented by measurements, tag keys and fields. Warning, may take a while.")
 
 	inspectReportTSMCommand.Flags().StringVarP(&inspectReportTSMFlags.orgID, "org-id", "", "", "process only data belonging to organization ID.")
+	inspectReportTSMCommand.Flags().StringVarP(&inspectReportTSMFlags.orgID, "org", "o", "", "process only data belonging to organization ID.")
```
Comment (dearyhud) [inline cmd/influx/inspect.go]: just caught this. will patch this up

File: cmd/influx/pkg.go [hunk 2]
```diff
@@ -108,11 +109,32 @@ func (b *cmdPkgBuilder) cmdPkgApply() *cobra.Command {
...
+			return fmt.Errorf("must specify org-id, or org name not both")
+		}
 		color.NoColor = !b.hasColor
+		var influxOrgID *influxdb.ID
 
-		influxOrgID, err := influxdb.IDFromString(b.orgID)
-		if err != nil {
-			return fmt.Errorf("invalid org ID provided: %s", err.Error())
+		if b.orgID != "" {
+			var err error
```
Comment (jsteenb2) [inline cmd/influx/pkg.go]: this isn't going to work, breaks the pattern for the builder...

Comment (jsteenb2) [reply cmd/influx/pkg.go]: any SVC dep needs to be provided through DI not dumped in here globally

needs fixup

Comment (jademcgough) [reply cmd/influx/pkg.go]: Thanks for catching, we'll fix in the morning.

**Pull request merged**